docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -568,6 +568,12 @@ NB: You must `REDIRECT_MACARON_LOG` and have `DISABLE_ROUTER_LOG` set to `false`
 
 - `SCHEDULE`: **@every 24h** : Interval as a duration between each synchronization, it will always attempt synchronization when the instance starts.
 
+### Cron - Prune hook_task Table (`cron.prune_hook_task_table`)
+
+- `ENABLED`: **true**: Enable service.
+- `RUN_AT_START`: **false**: Run prune hook_task at start time (if ENABLED).
+- `SCHEDULE`: **@every 24h**: Cron syntax for pruning hook_task table.
+
```

Comment:
Also `DefaultIsHookTaskPurgeEnabled` and `DefaultNumberWebhookDeliveriesToKeep`?

Reply:
those are set to defaults in the repository table for each repo and then can be overridden per repo via ui or db (see below). Do you think there would need to be a global override for those, which I guess is what putting it in the app.ini would accomplish?

![image](https://user-images.githubusercontent.com/32200834/89315167-27128e80-d640-11ea-98e5-f7f468766b3a.png)

Reply:
I mean we also need add document for the two new configuration items.

Reply:
Gotcha, sorry. I'm a little confused how to document in here, because the defaults cannot be overridden via the config file the way I implemented it. However, just looking through the file I see config items like [repository.pull-request] - DEFAULT_MERGE_MESSAGE_SIZE that makes it sound like you can change the default, but on a code search that's not used and repository.go just hardcodes the value.

It didn't feel right to do that so,I added a blurb about these values, but not as a separate lines. But if I've missed your point let me know, thanks!